### PR TITLE
Documentation edits

### PR DESCRIPTION
**Pull request by JennaPaikowsky**
Minor language edits based on documentation review.

### CODE REVIEW

File: doc/source/users_guide/analysis_setup.rst [hunk 1]
```diff
@@ -45,7 +45,7 @@ such as the variables it exposes and the regions on which they are available.
 
 The name of the created ``coupling_participant`` object is returned in each case.
 
-This code shows how you can capture the name in a variable to facilitate subsequent access
+This code shows how you can capture the name in a variable to provide subsequent access
```
Comment (iboyd-ansys) [inline doc/source/users_guide/analysis_setup.rst:48]: I'm a little unsure about "provide" here. I would say that it is the API itself that is providing access to the object. There are other ways to do it, but saving the name in a variable is just one way that makes it easy. Could we say something like "help with" or "assist with"?

**Review by iboyd-ansys — COMMENTED** [review]
Looks good, thanks. I just had one comment but am open to being persuaded.

Note that I will need to regenerate the examples and push to the PR before we can mark is as approved.

File: doc/source/users_guide/index.rst [hunk 2]
```diff
@@ -86,7 +86,7 @@ You can turn off output streaming using this code to call the ``end_output()`` m
 
 Exiting
 -------
-When you are finished with a PySystemCoupling session, it is advisable to end it cleanly using the
+When you are finished with a PySystemCoupling session, we recommend ending it cleanly using the
```
Comment (PipKat) [inline doc/source/users_guide/index.rst:89]:
```suggestion
When you are finished with a PySystemCoupling session, you should end it cleanly using the
```

File: doc/source/users_guide/index.rst [hunk 3]
```diff
@@ -86,7 +86,7 @@ You can turn off output streaming using this code to call the ``end_output()`` m
...
 ``exit()`` method. If you do not do this, PySystemCoupling will still attempt to clean up any active server sessions
```
Comment (PipKat) [inline doc/source/users_guide/index.rst:90]:
```suggestion
``exit()`` method. If you do not do this, PySystemCoupling still attempts to clean up any active server sessions
```
Use present tense.

File: doc/source/users_guide/analysis_setup.rst (hunk 1 above)
Comment (JennaPaikowsky) [reply doc/source/users_guide/analysis_setup.rst]: Changed to "assist with".

**Review by iboyd-ansys — APPROVED** [review]

**Comment by iboyd-ansys** [comment]
@JennaPaikowsky I am going to merge this without rerunning the examples. The reason being that the changes to the examples are quite small and it turns out that I need to make a new release in the next couple of days so I am going to have to re-run the examples then in any case. It will mean that the examples will be out of sync with your changes for a couple of days.

**Pull request merged**